src/components/scratch-image/scratch-image.jsx

```diff
@@ -136,6 +136,7 @@ ScratchImage.ImageSourcePropType = PropTypes.oneOfType([
 ]);
 
 ScratchImage.propTypes = {
+    src: PropTypes.string,
```

Comment:
@paulkaplan lint was complaining about `src` being missing from the props validation, so I added it. However, I'm not sure if src is actually used/needed (defined above on line 97). 
/cc @cwillisf